itowns/Viewer.js

```diff
@@ -210,7 +215,7 @@ class Viewer {
             opacity,
             style,
             zoom: {
-              min: layerName === 'Patches' ? this.zoomMinPatch : this.overviews.dataSet.level.min,
+              min: layerName === 'Patches' || layerName === this.alertLayerName ? this.zoomMinPatch - 1 : this.overviews.dataSet.level.min,
```

Comment:
Pourquoi "this.zoomMinPatch - 1" ?

Reply:
Pour limiter un trop grand clignotement (à cause du bug itowns) j'ai quand même limité le zoom sur la couche d'alerte (alors qu'il ne devras pas y avoir de limitation, d'où le -1 temporaire)

Reply:
Supprimer (c'était en lien avec les effets de clignotement)